Framework/Indexing/inc/MantidIndexing/IndexInfo.h

```diff
@@ -91,6 +100,9 @@ class MANTID_INDEXING_DLL IndexInfo {
 
   Kernel::cow_ptr<std::vector<specnum_t>> m_spectrumNumbers;
   Kernel::cow_ptr<std::vector<std::vector<detid_t>>> m_detectorIDs;
+
+  Kernel::cow_ptr<std::vector<Beamline::SpectrumDefinition>>
```

Comment:
Typically many workspaces have the same spectra (usually all workspaces for the same instrument), so we want to share as much as possible. What were your thoughts exactly?